bin/cosmos

```diff
@@ -253,6 +287,9 @@ if not ARGV[0].nil? # argument(s) given
   when 'generate'
     generate(ARGV[1..-1])
 
+  when 'check_limits'
+    check_limits(ARGV[1..-1])
```

Comment:
So the idea is that you run this against bin files to determine any limits violations in the bin files? How does this fit into a conops?

Reply:
This just provides a way for extracting the limits violations from a log file.  A useful detail we didn't have before.